modules/mobile/spec/models/adapters/appointments_vaos_v2_adapter_spec.rb

```diff
@@ -3,30 +3,42 @@
 require 'rails_helper'
```

Comment:
I do not recommend trying to read the diff because it will be unreadable. Read the raw file instead because this is a near complete rewrite.